pkg/resource/feature_group/manager_test_suite_test.go

```diff
@@ -0,0 +1,196 @@
+		return &output, nil
+	case "DeleteFeatureGroup":
+		var output svcsdk.DeleteFeatureGroupOutput
+		return &output, nil
+	case "DeleteFeatureGroupRequest":
+		var output svcsdk.DeleteFeatureGroupOutput
+		return &output, nil
+	case "DeleteFeatureGroupWithContext":
+		var output svcsdk.DeleteFeatureGroupOutput
+		return &output, nil
+	case "DescribeFeatureGroup":
+		var output svcsdk.DescribeFeatureGroupOutput
+		return &output, nil
+	case "DescribeFeatureGroupRequest":
+		var output svcsdk.DescribeFeatureGroupOutput
+		return &output, nil
+	case "DescribeFeatureGroupWithContext":
+		var output svcsdk.DescribeFeatureGroupOutput
+		return &output, nil
+	case "ListFeatureGroups":
+		var output svcsdk.ListFeatureGroupsOutput
+		return &output, nil
+	case "ListFeatureGroupsPages":
+		var output svcsdk.ListFeatureGroupsOutput
+		return &output, nil
+	case "ListFeatureGroupsRequest":
+		var output svcsdk.ListFeatureGroupsOutput
+		return &output, nil
+	case "ListFeatureGroupsWithContext":
+		var output svcsdk.ListFeatureGroupsOutput
+		return &output, nil
+	}
+	return nil, errors.New(fmt.Sprintf("no matching API name found for: %s", apiName))
+}
+
+func (d *testRunnerDelegate) Equal(a acktypes.AWSResource, b acktypes.AWSResource) bool {
+	ac := a.(*resource)
+	bc := b.(*resource)
+	// Ignore LastTransitionTime since it gets updated each run.
+	opts := []cmp.Option{cmpopts.EquateEmpty(), cmpopts.IgnoreFields(ackv1alpha1.Condition{}, "LastTransitionTime")}
+
+	if cmp.Equal(ac.ko.Status, bc.ko.Status, opts...) {
+		return true
+	} else {
+		fmt.Printf("Difference (-expected +actual):\n\n")
+		fmt.Println(cmp.Diff(ac.ko.Status, bc.ko.Status, opts...))
+		return false
+	}
+}
+
+// Checks to see if the given yaml file, with name stored as expectation,
+// matches the yaml marshal of the AWSResource stored as actual.
+func (d *testRunnerDelegate) YamlEqual(expectation string, actual acktypes.AWSResource) bool {
+     	// Get the file name of the expected yaml.
+	expectedYamlFileName := TestDataDirectory + "/" + expectation
+
+	// Build a tmp file for the actual yaml.
+	actualResource := actual.(*resource)
+	actualYamlByteArray, _ := yaml.Marshal(actualResource.ko)
+	actualYamlFileName := buildTmpFile("actualYaml", actualYamlByteArray)
+	defer os.Remove(actualYamlFileName)
+	if "" == actualYamlFileName {
+	  fmt.Printf("Could not create temporary actual file.\n")
+	  return false
+	}
+
+	// Replace Timestamps that would show up as different.
+	_, err := exec.Command("sed", "-r", "-i", ReplaceTimestampRegExp, actualYamlFileName).Output()
+	if !checkExecCommandError(err) {
+	    return false
+	}
+	
+	output,err := exec.Command("diff", "-c", expectedYamlFileName, actualYamlFileName).Output()
+	if !checkExecCommandError(err) {
+	   return false
+	}
+
+	if len(output) > 0 {
+	   actualOutput,_ := exec.Command("cat", actualYamlFileName).Output()
+	   fmt.Printf("\nActual Yaml File Instead of: " + expectedYamlFileName + "\n" + string(actualOutput) + "\n")
```

Comment:
i feel wording is kinda weird 

Can we just print : 
Expected Yaml File Name :  `Full path to file expectedYamlFileName` 
Actual Output Yaml : `string(actualOutput)` 
Diff : `string(output)`